### PR TITLE
test tool changes for bot-all-cards sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/bot-all-cards/nodejs/.localConfigs.testTool [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# A gitignored place holder file for local runtime configurations when debug in test tool
+BOT_ID=
```
Comment (Pawank-MSFT) [inline samples/bot-all-cards/nodejs/.localConfigs.testTool:2]: .localConfigs.testTool file needed or not?

File: samples/bot-all-cards/nodejs/.notification.testtoolstore.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (Pawank-MSFT) [inline samples/bot-all-cards/nodejs/.notification.testtoolstore.json:1]: .notification.testtoolstore file needed or not?

File: samples/bot-all-cards/nodejs/.teamsapptesttool.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=https://aka.ms/teams-app-test-tool-config/0.1.0/config.schema.json
+# Visit https://aka.ms/teams-app-test-tool-config-guide for details on this file
+
```
Comment (Pawank-MSFT) [inline samples/bot-all-cards/nodejs/.teamsapptesttool.yml:3]: .teamsapptesttool.yml file is needed or not?

File: samples/bot-all-cards/nodejs/teamsapp.yml [hunk 4]
```diff
@@ -6,4 +6,5 @@ version: v1.2
 additionalMetadata:
   sampleTag: Microsoft-Teams-Samples:bot-all-cards-nodejs
 
-environmentFolderPath: ./env
\ No newline at end of file
+environmentFolderPath: ./env
+projectId: 5e3de923-95a7-4199-8034-24a984ae3ddf
```
Comment (Pawank-MSFT) [inline samples/bot-all-cards/nodejs/teamsapp.yml:10]: remove ProjectID and other keys

**Review by Pawank-MSFT — COMMENTED** [review]
some comments please fix

**Review by Mohammed-MSFT — APPROVED** [review]
Changes Looks Fine, Approving It

File: samples/bot-all-cards/nodejs/.teamsapptesttool.yml (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/bot-all-cards/nodejs/.teamsapptesttool.yml:3]: yes this file is needed

File: samples/bot-all-cards/nodejs/.notification.testtoolstore.json (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/bot-all-cards/nodejs/.notification.testtoolstore.json:1]: yes this file is needed

File: samples/bot-all-cards/nodejs/.localConfigs.testTool (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/bot-all-cards/nodejs/.localConfigs.testTool]: yes this file is needed

File: samples/bot-all-cards/nodejs/teamsapp.yml (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/bot-all-cards/nodejs/teamsapp.yml]: removed

File: samples/bot-all-cards/nodejs/.teamsapptesttool.yml (hunk 3 above)
Comment (frankqianms) [reply samples/bot-all-cards/nodejs/.teamsapptesttool.yml:3]: do we really need these dummy users' information for this sample? if not, we can remove this file and the bot can still work.

**Review by frankqianms — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build pipeline - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8387901072

**Pull request merged**